test/erlfmt_format_SUITE.erl

```diff
@@ -2591,12 +2608,53 @@ receive_expression(Config) when is_list(Config) ->
         "end\n",
         "receive\n"
         "    1 -> ok\n"
-        "%% after receive\n"
-        "\n"
+        "    %% after receive\n"
         "after 0 ->\n"
         "    ok\n"
         "    %% after after for receive\n"
         "end\n"
+    ),
+    ?assertFormat(
+        "receive\n"
+        "    1 -> ok\n"
+        "    %% after receive\n"
+        "after\n"
+        "    %% before zero\n"
+        "    0 -> ok\n"
+        "    %% after after for receive\n"
+        "end\n",
+        "receive\n"
+        "    1 -> ok\n"
+        "    %% after receive\n"
+        "after\n"
+        "    %% before zero\n"
+        "    0 -> ok\n"
+        "        %% after after for receive\n"
+        "end\n"
+    ),
+    ?assertFormat(
+        "receive\n"
+        "    1 -> ok\n"
+        "    %% after receive\n"
+        "after\n"
+        "    %% before zero\n"
+        "    0 -> a(\n"
+        "        b,\n"
+        "        c\n"
+        "    )\n"
+        "    %% after after for receive\n"
+        "end\n",
+        "receive\n"
+        "    1 -> ok\n"
+        "    %% after receive\n"
+        "after\n"
+        "    %% before zero\n"
+        "    0 -> a(\n"
+        "            b,\n"
+        "            c\n"
+        "        )\n"
+        "        %% after after for receive\n"
```

Comment:
Oh, I missed this one before.

I don't think we should do that. We don't do it for regular clauses. Is there a particular reason why here it's desirable?

Reply:
This is a new test

Reply:
You didn't miss it before.

Reply:
I don't think this is correct. i just tried it, and for example with a concat node as the value, it does something like this:

```erl
receive
after
    %% foo
    0 -> "abc"
    "def"
end.
```

Reply:
Good catch. I have added it as a test.

Reply:
These are both fixed now, but see new issues below.